src/js/widgets/RadioWidget.jsx

```diff
@@ -25,7 +25,7 @@ export default function RadioWidget({
         const radioButton = (
           <div className="form-radio-buttons" key={option.value}>
             <input type="radio"
-              autoComplete="false"
+              autoComplete={options.autocomplete || null}
```

Comment:
I'm pretty sure this will throw an axe error like the checkboxes. Please advise if you'd like confirmation and I can mock it on Codepen or other site for testing.

Reply:
The [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) aren't clear to me on whether it's valid for a radio or not. It seems to say under `checked` that you can use it.